src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageViewModel.cs

```diff
@@ -30,11 +33,15 @@ internal class RPackageViewModel : BindableBase, IRPackageViewModel {
                 Suggests = package.Suggests.NormalizeWhitespace(),
                 License = package.License.NormalizeWhitespace(),
                 NeedsCompilation = package.NeedsCompilation != null && !package.NeedsCompilation.EqualsIgnoreCase("no"),
-                Repository = package.Repository,
+                RepositoryUri = uri,
+                RepositoryText = uri != null ? null : package.Repository,
             };
         }
 
         public static RPackageViewModel CreateInstalled(RPackage package) {
+            Uri uri = null;
+            Uri.TryCreate(package.Repository, UriKind.Absolute, out uri);
```

Comment:
You get null URI so text will be used which is also null. You get Repositiry: and then empty